website/content/en/preview/concepts/disruption.md

```diff
@@ -198,6 +197,39 @@ Karpenter enables this feature by watching an SQS queue which receives critical
 
 To enable interruption handling, configure the `--interruption-queue` CLI argument with the name of the interruption queue provisioned to handle interruption events.
 
+### Node Auto Repair 
+
+Node Auto Repair is a feature that automatically identifies and replaces unhealthy nodes in your cluster, helping to maintain overall cluster health. Nodes can experience various types of failures affecting their hardware, file systems, or container environments. These failures may be surfaced through node conditions such as network unavailability, disk pressure, memory pressure, or other conditions reported by node diagnostic agents. When Karpenter detects these unhealthy conditions, it automatically replaces the affected nodes based on cloud provider-defined repair policies. Once a node has been in an unhealthy state beyond its configured toleration duration, Karpenter will forcefully terminate the node and its corresponding NodeClaim, bypassing the standard drain and termination grace period procedures to ensure swift replacement of problematic nodes. This ensures your cluster remains in a healthy state with minimal manual intervention, even in scenarios where normal node termination procedures might be impacted by the node's unhealthy state.
+
+To enable Node Auto Repair: 
+  1.  Ensure you have a [Node Monitoring Agent](https://docs.aws.amazon.com/en_us/eks/latest/userguide/node-health.html) deployed(e.g., Node Problem Detector)
+  2.  Enable the feature flag: NodeRepair=true
+  3.  The feature will automatically begin monitoring nodes based on your cloud provider's repair policies
+
```

Comment:
Should we mention anything about the rate limiting that we implemented to ensure that we don't terminate too many of these nodes at once?